### PR TITLE
Add EBS-capable task definitions

### PR DESCRIPTION
**Pull request by alexwlchan**
For https://github.com/wellcomecollection/loris-infrastructure/pull/4

### CODE REVIEW

File: task_definition/container_with_sidecar/task_definition.tf [hunk 1]
```diff
@@ -4,7 +4,17 @@ module "task_role" {
   task_name = var.task_name
 }
 
+# The presence of two almost identical blocks of config is working
```
Comment (kenoir) [inline task_definition/container_with_sidecar/task_definition.tf]: This is no longer true: https://www.terraform.io/docs/configuration/expressions.html#dynamic-blocks

Comment (alexwlchan) [reply task_definition/container_with_sidecar/task_definition.tf]: Ooh, interesting.

Comment (jamesgorrie) [reply task_definition/container_with_sidecar/task_definition.tf]: 🚀

Comment (alexwlchan) [reply task_definition/container_with_sidecar/task_definition.tf]: Hmm. Having read how dynamic blocks work, I’m inclined to leave it as-is.

You can do dynamic blocks, but it’s a bit fiddly – we don’t want “0 to N”, we want “on or off”. We could make it work, but you're shoe-horning a boolean into a loop.

How would you feel about me leaving a comment “we looked at dynamic blocks, weren't right last time we looked, but keep an eye on them in future”?

File: service/service.tf [hunk 2]
```diff
@@ -37,45 +19,32 @@ resource "aws_ecs_service" "service" {
...
-
-  service_registries {
-    registry_arn = aws_service_discovery_service.service_discovery.arn
-  }
-
-  load_balancer {
-    target_group_arn = var.target_group_arn
-    container_name   = var.container_name
-    container_port   = var.container_port
+  # This is a kludge around Terraform's "dynamic blocks".
```
Comment (kenoir) [inline service/service.tf:26]: 😂

Comment (kenoir) [reply service/service.tf:26]: Kludges all the way down.

**Comment by alexwlchan** [comment]
Taking Robert’s kludges comment as tacit approval, merging!

**Pull request merged**